nebari/template/stages/02-infrastructure/aws/main.tf

```diff
@@ -5,6 +5,14 @@ data "aws_availability_zones" "awszones" {
   }
 }
 
+
+locals {
+  # Only override_network if both existing_subnet_ids and existing_security_group_id are not null.
+  override_network  = (var.existing_subnet_ids != null) && (var.existing_security_group_id != null)
+  subnet_ids        = local.override_network ? var.existing_subnet_ids : module.network[0].subnet_ids
+  security_group_id = local.override_network ? var.existing_security_group_id : module.network[0].security_group_id
```

Comment:
Is there a reason why this being indexed `[0]`? Looking at [line 60](https://github.com/nebari-dev/nebari/pull/1807/files#diff-afa6624e083ba0880d58a57da8bc5e580b4e915f5bd966e9dc802e240afd3cd4L60), it appears it wasn't before.

Reply:
Adding count under module.network makes the index required now